modules/metering-install-verify.adoc

```diff
@@ -2,44 +2,71 @@
 //
 // * metering/metering-installing-metering.adc
 
-[id="metering-install-verify_{context}"] 
-= Verify Installation 
+[id="metering-install-verify_{context}"]
+= Verifying the metering installation
 
-To verify your metering installation you can check that all the required Pods have been created, and check that your report data sources are beginning to import data.
+You can verify the metering installation by performing any of the following:
```

Comment:
Add "... the following **steps**:" or other word to make the introductory text a complete sentence.